src/analysis/special_functions/trigonometric.lean

```diff
@@ -26,7 +26,7 @@ Many basic inequalities on trigonometric functions are established.
 The continuity and differentiability of the usual trigonometric functions are proved, and their
 derivatives are computed.
 
-* `polynomial.chebyshev₁_complex_cos`: the `n`-th Chebyshev polynomial evaluates on `complex.cos θ`
+* `polynomial.chebyshev_T_complex_cos`: the `n`-th Chebyshev polynomial evaluates on `complex.cos θ`
```

Comment:
```suggestion
* `polynomial.chebyshev_T.complex_cos`: the `n`-th Chebyshev polynomial evaluates on `complex.cos θ`
```

Reply:
@eric-wieser can you explain why you suggest this? I would have expected
```suggestion
* `polynomial.chebyshev.T_complex_cos`: the `n`-th Chebyshev polynomial evaluates on `complex.cos θ`
```

Reply:
It is because I cannot type :)